README.md

```diff
@@ -50,3 +50,11 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+The flask app has now been altered so that it is now partially integrated to the Trello Website API. To run the altered app one needs:
+
+-A Trello Account to access the Trello API (This is free)
+
+-To set up the Trello enviromental variables (Key, Token, List IDs etc) either by the altering the DevOps-Course-Starter/.env file or creating a seperate config file with those variables inside
```

Comment:
This line is untrue - your code fetches the variables with `from todo_app.trello_config import KEY, TOKEN, TODOID, DONEID`

So in reality, the reader needs to set up a file called trello_config.py in the correct folder and containing those exact variables.

Reply:
Hi Hugh. Have updated it on the module 3 branch but instead it's the fact that the variables are now only reflected in the env file and now is only called by os.getenv()